addons/web/static/src/core/hotkeys/hotkey_service.js

```diff
@@ -164,8 +164,11 @@ export const hotkeyService = {
             for (const el of getVisibleElements(ui.activeElement, "[data-hotkey]:not(:disabled)")) {
                 const hotkey = el.dataset.hotkey;
                 const overlay = document.createElement("div");
-                overlay.className = "o_web_hotkey_overlay";
-                overlay.appendChild(document.createTextNode(hotkey.toUpperCase()));
+                const overlay_kbd = document.createElement("kbd");
+                overlay.className = "o_web_hotkey_overlay position-absolute d-flex justify-content-center align-items-center m-0 bg-black-50 h6";
+                overlay_kbd.className = "bg-light border border-dark small";
+                overlay.appendChild(overlay_kbd);
+                overlay_kbd.appendChild(document.createTextNode(hotkey.toUpperCase()));
```

Comment:
To use  a `<kbd>` tag is semantically correct and improves readability while keeping `o_web_hotkey_overlay` with a relatively low background-opacity... but I'm not sure if all the classes you applied at the `<kbd>` are worth (`bg-light border border-dark small`)... why not keeping `<kbd>` default design (screenshoot, "A" vs "B")?

I let you choose, both are good to me.



![image](https://user-images.githubusercontent.com/8558351/152184530-2749281b-adaa-43aa-9e97-642931e7bb03.png)

Reply:
Hello @stefanorigano 👋 ,

After a vote with the design team, the winner is ... the white version ! 👍

Reply:
the people has spoken, ok 👍